### ISSUE TITLE
Error reading file with just one response

### ISSUE THREAD

**Issue by sloboy** [YukiUchino/PressGaneyPatientSatisfactionParserLite#1]
Sorry if this is necroing, but I have the same problem to solve with PG survey files.

When the XML file has a demographics section with only one response, the program errors out complaining of using a string instead of an integer as an index.  I think the issue is as seen in these links, and was able to run the file through the program when I added force_list=response as seen here:
 doc = xmltodict.parse(fd.read(),process_namespaces=True,force_list=['RESPONSE'])

I will send a pull request soon that fixes this and a couple of other things that I noticed.

https://stackoverflow.com/questions/37207353/xmltodict-does-not-return-a-list-for-one-element
https://github.com/martinblech/xmltodict/issues/14

**Comment by YukiUchino** [comment]
Great, thanks! I haven't run the code in awhile, so I'm sure there are a
number of things that have cropped up when using the code in the wild.
Thanks for addressing them, I'll be on the lookout for the pull request!

**Comment by puddlejumper90** [comment]
Are you open to other people helping out with this project? There are probably a ton of hospitals that would benefit from this project. I have been trying to parse XML Press Ganey data as well.

**Comment by YukiUchino** [comment]
@puddlejumper90, yes I am open to other people helping out on this project. If you have a fix, by all means please feel free to either submit a pull request, or message me a detailed description of the fix I can implement on your behalf.
